docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+services:
+  db:
+    image: "postgres:9.5"
```

Comment:
There would need to be a port mapping here to allow a db client outside of the container to connect to it, right?